neuralmonkey/nn/projection.py

```diff
@@ -64,3 +67,17 @@ def maxout(inputs, size, scope="MaxoutProjection"):
 
         reshaped = tf.reshape(maxpooled, [-1, size])
         return reshaped
+
+
+def multilayer_projection(input_, layer_sizes, activation=tf.nn.relu,
```

Comment:
Achjo, včera jsem to sáhodlouze komentoval a teď tady ten komentář vůbec nevidím. MLP je klasifikátor, tj. má na konci softmax. Tohle je jenom projekce. Třída MLP už teď tuhle funkci volá.